resources/content/cattle-global.properties

```diff
@@ -38,6 +38,10 @@ machine.driver.amazonec2=true,local://,
 machine.driver.azure=true,local://,
```

Comment:
All changes unneeded due to https://github.com/rancher/cattle/pull/1994#issuecomment-248626932